### ISSUE TITLE
Move json converters into subpackage

### ISSUE THREAD

**Issue by armintaenzertng** [spdx/tools-python#405]
The `jsonschema` package is currently rather full and incomprehensible. Introducing a new subpackage `conversion` can change that.

**Comment by HarshvMahawar** [comment]
So we have to introduce conversion inside the jsonschema package or at another place?

**Comment by armintaenzertng** [comment]
I would suggest moving every file in the `jsonschema` package ending in `converter.py` into a subpackage `conversion` and everything ending in `properties.py` into a subpackage `properties`.
Both of these subpackages can be direct children of the `jsonschema` package.

**Comment by HarshvMahawar** [comment]
But after doing that I will also have to change the dependencies
( from spdx.jsonschema import annotation_converter.py -> from spdx.jsonschema.converter import annotation_converter.py )  in each file.

**Comment by armintaenzertng** [comment]
Yes, but this should be handled automatically by your IDE, which should scan the whole code for these imports and adapt them accordingly.

**Comment by HarshvMahawar** [comment]
yes, so all I have to do is transfer files and that's it? ( __init__ and optional_utils will remain in jsonschema only) .How should I test the changes (pytest n all)

**Comment by armintaenzertng** [comment]
Yeah, it's not really a big issue, just a minor refactoring, which is why there are no new tests to write for this.

The `optional_utils` module is only used in the converters, so I suggest moving it to `conversion`, too.

**Comment by armintaenzertng** [comment]
Looks good, don't hesitate to open a PR (you can always adapt it if necessary)! :)

**Comment by HarshvMahawar** [comment]
I have opened it but it seems all the install_and_test tests were failed, what could be the reason behind it? could it be that my remote repo is behind the spdx:main

**Comment by armintaenzertng** [comment]
The issue occurs because you did not commit the changes to the imports that we discussed earlier.

**Comment by HarshvMahawar** [comment]
oh...I totally forgot it, thanks for pointing it out

**Comment by armintaenzertng** [comment]
Have look the [contribution guideline](https://github.com/spdx/tools-python/blob/main/CONTRIBUTING.md), especially at points 6 and 7 under the heading "Development process". There you find the commands to test your local code without the need to have to wait for the GitHub Action run.

**Comment by armintaenzertng** [comment]
Have you tried running `pytest` on your local machine? This will give you an overview of all the failing tests, which you can then have a look at separately to figure out what's going wrong.

**Comment by armintaenzertng** [comment]
It looks like your IDE did not recognize the mock paths, like for example [here](https://github.com/spdx/tools-python/pull/575/files#diff-6ece6c48dd891d3e153d3f71a7aeb7fb532f1ab7680af50a5825c42b0bc9cf2fR34). Those have to be adapted, too.

**Comment by HarshvMahawar** [comment]
yes, I am researching some way to solve this problem otherwise I will have to do it by hard way(manually)
